driver/docker-container/driver.go

```diff
@@ -44,18 +44,19 @@ type Driver struct {
 
 	// if you add fields, remember to update docs:
 	// https://github.com/docker/docs/blob/main/content/build/drivers/docker-container.md
-	netMode       string
-	image         string
-	memory        opts.MemBytes
-	memorySwap    opts.MemSwapBytes
-	cpuQuota      int64
-	cpuPeriod     int64
-	cpuShares     int64
-	cpusetCpus    string
-	cpusetMems    string
-	cgroupParent  string
-	restartPolicy container.RestartPolicy
-	env           []string
+	netMode           string
+	image             string
+	memory            opts.MemBytes
+	memorySwap        opts.MemSwapBytes
+	cpuQuota          int64
+	cpuPeriod         int64
+	cpuShares         int64
+	cpusetCpus        string
+	cpusetMems        string
+	cgroupParent      string
+	restartPolicy     container.RestartPolicy
+	env               []string
+	defaultLoadTagged bool
```

Comment:
I think `defaultLoad` is enough to be consistent with `BUILDX_NO_DEFAULT_LOAD` env var: https://docs.docker.com/build/building/variables/#build-tool-configuration-variables

Reply:
Thanks! Only loading tagged images was supposed to be a latency optimization as a pull from a remote driver can be slow.
But I agree that consistency is important. Updated.